dashboard/src/components/user/UserNav.vue

```diff
@@ -24,7 +24,6 @@
 			<div>Hello, {{ user.name }}!</div>
 		</a>
 		<div class="navbar-dropdown">
-			<hr class="navbar-divider" />
```

Comment:
Is this needed at all?

Reply:
probably left over from when I had added more than one element here